libs/blocks/media/media.css

```diff
@@ -103,13 +106,93 @@ div[class*="-up"] .media .foreground > .media-row {
   padding: 0 0 var(--spacing-m);
 }
 
+.media .default-list {
+  padding-left: 20px;
+  margin-top: var(--spacing-xs);
+  margin-bottom: var(--spacing-xs);
+}
+
+.media li::before {
+  content: "";
+  display: inline-block;
+  margin-left: 3px;
+}
+
+.media.checklist li {
+  background: url("/libs/img/ui/checkmark-green.svg") no-repeat 5px 5px transparent;
+  list-style-type: none;
+  margin-left: -25px;
+  padding: 0 0 1px 29px;
+  vertical-align: middle;
+}
+
+.media .subcopy {
+  font-size: var(--type-heading-xs-size);
+  line-height: var(--type-body-s-lh);
+  margin-top: var(--spacing-xs);
+  color: #2C2C2C;
```

Comment:
```suggestion
```

This isn't needed. Text default color is already set at this hex?

Reply:
Yes, you are right @ryanmparrish . I have removed this line of code